CHANGELOG.md

```diff
@@ -4,14 +4,38 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.4.0 (2021-07-26)
+This release adds new experimental features since the 2.3 release.
```

Comment:
```suggestion
This release adds new experimental features since the 2.3 release.
Experimental schemas enable us to validate new functionality and get feedback from real users, while allowing developers to test the latest and greatest of what TimescaleDB has to offer.
```